js/data.js

```diff
@@ -1,8 +1,35 @@
 /* exported data */
-
+var $form = document.querySelector('#entry-form');
+var $title = document.querySelector('.title');
+var $photoUrl = document.querySelector('.photo-url');
+var $notes = document.querySelector('.notes');
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+var previousData = localStorage.getItem('code-journal-entries');
+
+if (previousData !== null) {
+  data.entries = JSON.parse(previousData);
+}
+
+function checkSubmit(event) {
+  event.preventDefault();
+  var submission = {
+    Title: $title.value,
+    'Photo URL': $photoUrl.value,
+    Notes: $notes.value,
+    nextEntryId: data.nextEntryId
```

Comment:
Why is this named `nextEntryId`? That property is named that way on the data model because its value is meant to be the "next" id to use. On this object, it is just the id for the entry that was just created, this property should be named `entryId`